src/Ledger/Ratify.lagda

```diff
@@ -200,17 +200,25 @@ acceptedStake r cc dists votes =
 totalStake : GovRole → ℙ VDeleg → StakeDistrs → (VDeleg ⇀ Vote) → Coin
 totalStake r cc dists votes = Σᵐᵛ[ x ← getStakeDist r cc dists ∣ votedAbstainHashes votes r ᶜ ᶠᵐ ] x
 
+activeVotingStake : ℙ VDeleg → StakeDistrs → (VDeleg ⇀ Vote) → Coin
+activeVotingStake cc dists votes = Σᵐᵛ[ x ← getStakeDist DRep cc dists ∣ dom (votes ˢ) ᶜ ᶠᵐ ] x
```

Comment:
`cc` is a bit misleading here, maybe `vdelegs` would be better?

Reply:
This is probably something to refactor. Note that the name `cc` is also used in the previous definition, and this argument is just used to pass the constitutional committee (anything else in there is ignored).